src/interbank/interbank.service.ts

```diff
@@ -73,13 +86,21 @@ export class InterbankService {
       throw e;
     }
     let interbankList;
+    let res = {};
```

Comment:
**suggestion**: remove this

Reply:
you mean ={} or the whole line

Reply:
Use `const` here: https://github.com/phineasla/4.1-React-Karma-BE/pull/10#discussion_r1062126977